src/test/java/me/karakelley/tictactoe/UITest.java

```diff
@@ -0,0 +1,15 @@
+package me.karakelley.tictactoe;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class UITest {
+
+    @Test
+
+    public void testMessage() {
```

Comment:
I liked the first test you had more (with the exception of `System.setOut`. I'll be in the office this morning so I'll just explain why there.